cmd/crc/cmd/console.go

```diff
@@ -44,6 +46,9 @@ func showConsole(client *daemonclient.Client) (*client.ConsoleResult, error) {
 
 func runConsole(writer io.Writer, client *daemonclient.Client, consolePrintURL, consolePrintCredentials bool, outputFormat string) error {
 	result, err := showConsole(client)
+	if err == nil && result.ClusterConfig.ClusterType == preset.Microshift {
+		err = fmt.Errorf("error : this option is not supported for microshift preset. Please refer to the documentation for a list of supported presets for using this option")
```

Comment:
it might be better to tell user which preset it support

```
err = fmt.Errorf("error : this option is only supported for %s and %s preset", preset.OpenShift, preset.OKD)
```

Reply:
Thanks, I've updated it as per your suggestion.